app.rb

```diff
@@ -0,0 +1,92 @@
+require_relative 'lib/database_connection'
+require_relative 'lib/product_repository'
+require_relative 'lib/order_repository'
+
+# We need to give the database name to the method `connect`.
+DatabaseConnection.connect('shop_manager')
+
+class Application 
+  
+
+  def initialize(database, io, order_repository, product_repository)
+
+    DatabaseConnection.connect(database)
+    @io = io
+    @order_repo = order_repository
+    @product_repo = product_repository  
+
+  end
+
+  def run 
+    
+    @io.puts "Welcome to the shop management program! \n\n"
+    @io.puts "What do you want to do?"
+    @io.puts "1 - List all shop items"
+    @io.puts "2 - Create a new item"
+    @io.puts "3 - List all orders"
+    @io.puts "4 - Create a new order"
+    user = @io.gets.chomp 
+
+    case user
+
+    when "1"
+
+      repo = @product_repo.all
+      @io.puts "This is the list of products we sell: "
+      repo.each do |inventory|
+       
+        @io.puts "##{inventory.id} - #{inventory.name} - Price: #{inventory.unit_price} - Quantity: #{inventory.quantity}"
+      end
```

Comment:
I like the case statements. Could also create separate methods that do one thing e.g. one method that does line 36-39 (basically the formatting)